Makefile

```diff
@@ -16,27 +16,46 @@ Debug:
 
 Debug/Makefile: Debug config
 ifndef INSTALL_PREFIX
-	(cd Debug && cmake .. -DCMAKE_BUILD_TYPE=Debug)
+	cd Debug && NUM_DEVS=${NUM_DEVS} DEV_NAMES=${DEV_NAMES} cmake .. -DCMAKE_BUILD_TYPE=Debug
```

Comment:
The parenthesis are required to launch the command in a sub-shell that won't affect the working directory of the main one.  Please leave them.